llvm/include/llvm/CodeGen/AccelTable.h

```diff
@@ -360,6 +360,49 @@ class DWARF5AccelTable : public AccelTable<DWARF5AccelTableData> {
     unsigned Index;
     DWARF5AccelTableData::AttributeEncoding Encoding;
   };
+  union AbbrevDescriptor {
+    struct {
+      uint32_t CompUnit : 1;
+      uint32_t TypeUnit : 1;
+      uint32_t DieOffset : 1;
+      uint32_t Parent : 2;
+      uint32_t TypeHash : 1;
+      uint32_t Tag : 26;
```

Comment:
the tag only requires a uint16_t. Can we make this 16 bits instead of 26?

Or maybe this can be:
```
union AbbrevDescriptor {
  struct {
    uint16_t CompUnit : 1;
    uint16_t TypeUnit : 1;
    uint16_t DieOffset : 1;
    uint16_t Parent : 2;
    uint16_t TypeHash : 1;
    uint16_t Tag;
  } Bits;
  uint32_t Value = 0;
};
```

Reply:
Does it matter?